xilem/src/view/button.rs

```diff
@@ -50,18 +51,34 @@ where
     type ViewState = ();
 
     fn build(&self, ctx: &mut ViewCtx) -> (Self::Element, Self::ViewState) {
-        ctx.with_leaf_action_widget(|ctx| ctx.new_pod(widget::Button::new(self.label.clone())))
+        ctx.with_leaf_action_widget(|ctx| {
+            ctx.new_pod(widget::Button::from_label(
+                widget::Label::new(self.label.label.clone())
```

Comment:
```suggestion
                // TODO: Use `Label::build` here - currently impossible because `Pod` uses `WidgetPod` internally
                widget::Label::new(self.label.label.clone())
```